src/leader.py

```diff
@@ -31,6 +32,8 @@ class Leader:
 
     STRAIGHT_ANGLE = 90
     """The angle that the hardware associates as straight."""
+    DEMO_TURN_TIME = 2
```

Comment:
Can we make this 1 second? I feel like that's enough, but we can tune it later if it needs to be 1.5 or 2.

Reply:
ok, will do.